pkg/nodeconfig/payload/payload.go

```diff
@@ -56,6 +56,7 @@ const (
 	NetworkConfigurationScript = payloadDirectory + "/generated/network-conf.ps1"
 	// HybridOverlayName is the name of the hybrid overlay executable
 	HybridOverlayName = "hybrid-overlay-node.exe"
+	CSIProxyPath      = payloadDirectory + "csi-proxy/csi-proxy.exe"
```

Comment:
Please add a comment. You have inserted this inbetween two hybrid-overlay constants. Please move it elsewhere. Ideally this list should be alphabetical but that's a lost cause 😞